### PR TITLE
Wallet Empty Home UI

### LINKED ISSUE #16922: Wallet Home Page - Empty State
Implement the empty home page for wallet as in the designs: https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1-46866&mode=design&t=nqgryfqoXvUt9deI-0

Dependency: https://github.com/status-im/status-mobile/pull/16915

### PR DESCRIPTION
**Pull request by OmarBasem**
fixes: https://github.com/status-im/status-mobile/issues/16922

This PR implements UI for wallet empty home tab. This PR implements UI only. It does not implement any functionality or re-frame/status-go logic.

This PR adds a file `temp` next to the view file to contain temporary code to better separate between what's temporary and what's not.

The navigation menu can be opened by long-pressing on the illustration banner.

[Designs.](https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=1-46866&mode=design&t=nqgryfqoXvUt9deI-0)

Demo: 

https://github.com/status-im/status-mobile/assets/29354102/22376240-7736-44a4-91fa-48242a4fa201

### CODE REVIEW

**Review by rasom — APPROVED** [review]

File: src/status_im2/contexts/wallet/home/view.cljs [hunk 1]
```diff
@@ -1,38 +1,65 @@
...
+    (fn []
+      [rn/view
+       {:style {:margin-top top
+                :flex       1}}
+       [common.home/top-nav {:type :grey}]
+       [quo/wallet-overview temp/wallet-overview-state]
+       [rn/pressable
+        {:style         style/illustration-container
+         :on-long-press #(rf/dispatch [:show-bottom-sheet {:content temp/wallet-temporary-navigation}])}
+        [quo/text {:style {:color colors/white}} "Illustration here"]]
```
Comment (briansztamfater) [inline src/status_im2/contexts/wallet/home/view.cljs]: Probably this should be a wallet graph component in empty state (`:time-frame :empty`)

Comment (OmarBasem) [reply src/status_im2/contexts/wallet/home/view.cljs]: That's right @briansztamfater, fixed it 👍

**Comment by OmarBasem** [comment]
Hey @Francesca-G, 
a design review please :)

You can navigate to the new wallet tab by long pressing on the wallet tab icon.

**Review by briansztamfater — APPROVED** [review]

**Review by smohamedjavid — APPROVED** [review]
🚀

**Comment by Francesca-G** [comment]
@OmarBasem I was about to review the PR but after generating new keys the app crashes and when I try to reopen it, it crashes right away (I'm on iOS iPhone 13).

Please let me know if it's a problem on my side 🙏

**Comment by OmarBasem** [comment]
@Francesca-G no it's not a problem from your side. That was a recently introduced bug. Can you try this build please: https://i.diawi.com/b1YrYp

**Comment by Francesca-G** [comment]
> @Francesca-G no it's not a problem from your side. That was a recently introduced bug. Can you try this build please: https://i.diawi.com/b1YrYp

this one worked but I don't think I'm seeing the correct version of the Wallet home I'm supposed to review:

091F21A0-D6BD-4AA8-8EEE-E6DCB56A8C90_1_102_o

**Comment by OmarBasem** [comment]
@Francesca-G tap and hold on the wallet tab to navigate to the new wallet screen

**Review by Francesca-G — APPROVED** [review]
Here's the [design review](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=4291%3A77126&mode=design&t=ypOkUqK2l9Rn0o75-1), with minor fixes :)

**Comment by OmarBasem** [comment]
@Francesca-G thank you :)

**Pull request merged**